framework/include/media/MediaPlayer.h

```diff
@@ -191,7 +200,7 @@ class MediaPlayer
 	/**
 	 * @brief MediaPlayer operator==
 	 * @details @b #include <media/MediaPlayer.h>
-	 * This function is a synchronous api
+	 * This function is a synchronous API
```

Comment:
All the corrections can be added as a seperate commit.

Reply:
Okay, thanks. I'll do so from now on.